css/index.css

```diff
@@ -1,3 +1,382 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
 
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+.container {
+	width: 800px;
+	margin: auto;
+	 
+}
+
+@media (min-width: 501px) and (max-width: 800px) {
+    .container {
+        width: 100%;
+    }
+}
+
+@media ( max-width: 500px ) {
+    .container {
+        width: 100%;
+        margin: 10px;
+        margin-left: 20px;
+    }
+}
+
+.nav-bar {
+	display: flex;
+	justify-content: space-around;
+	width: 800px;
+	margin: 20px auto;
+	flex-direction: row-reverse;
+	align-items: center; 
+}
+
+@media (min-width: 501px) and (max-width: 800px) {
+    .nav-bar {
+        flex-wrap: wrap;
+        flex-direction: column;
+        width: 100%;
+    }
+}
+
+@media ( max-width: 500px ) {
+    .nav-bar {
+        flex-wrap: wrap;
+        flex-direction: column;
+        /* margin: 20px auto; */
+        padding: 0;
+        width: 100%;
+        border-bottom: 1px solid black;
+    }
+}
+
+nav {
+    display: flex;
+	justify-content: space-around;
+	width: 800px;
+	margin: 20px auto;
+	flex-direction: row;
+	align-items: center; 
+}
+
+@media (min-width: 501px) and (max-width: 800px) {
+
+    nav {
+        width: 100%;
+    }
+}
+
+@media ( max-width: 500px ) {
+    
+    nav {
+        flex-direction: column;
+        justify-content: space-evenly;
+        margin-top: 20px;
+        margin-bottom: 0;
+        /* padding: 30px 0%; */
+        width: 100%;
+        text-align: center;
+
+    }
+}
+
+nav a {
+    padding: 20px auto;
+	width: 80px;
```

Comment:
You should try to remove this `width: 80px` property, it should make your a tags in your navigation to be center in the 800px media query and it shouldn't affect your other views